contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,72 @@
+# See readme.md for instructions on running this code.
```

Comment:
I think you put your code in a separate folder right before the submission of the PR without test running it afterwards - if you try to run the code from the `DefineBot` folder, you get an error `Sorry, we will only import code from contrib_bots/lib.` and the bot won't run.